repose-aggregator/components/filters/rackspace-identity-basic-auth/src/main/scala/org/openrepose/filters/rackspaceidentitybasicauth/RackspaceIdentityBasicAuthHandler.scala

```diff
@@ -52,44 +54,38 @@ class RackspaceIdentityBasicAuthHandler(basicAuthConfig: RackspaceIdentityBasicA
     if (!httpServletRequest.getHeaderNames.asScala.toList.contains(X_AUTH_TOKEN)) {
       withEncodedCredentials(httpServletRequest) { encodedCredentials =>
         Option(datastore.get(TOKEN_KEY_PREFIX + encodedCredentials)) match {
-          case Some(token) => {
-            val tokenString = token.toString()
+          case Some(token) =>
+            val tokenString = token.toString
             filterDirector.requestHeaderManager().appendHeader(X_AUTH_TOKEN, tokenString)
-          }
-          case None => {
+          case None =>
```

Comment:
so yeah this is the case i'm talking about, they are way nested and should probably get scoped with parens to make it clear. But honestly this is a cluster and should have some methods extracted out of it explaining what it does, because it's to large to keep context for.